carvis-web/src/server/utils/uber-helper.js

```diff
@@ -1,34 +1,34 @@
-var fetch = require('node-fetch');
-var uberMethods = require('./uberPrivateMethods');
-var APItoken = require('./../../../secret/config.js')
-  .CARVIS_API_KEY;
-var APIserver = require('./../../../secret/config.js')
-  .CARVIS_API;
-var baseURL = 'https://cn-sjc1.uber.com';
+import fetch from 'node-fetch';
+import { login, confirmPickup } from './../utils/uberPrivateMethods.js';
+const baseURL = 'https://cn-sjc1.uber.com';
+const APItoken = !process.env.PROD ? require('./../../../secret/config.js')
+  .CARVIS_API_KEY : null;
+const APIserver = !process.env.PROD ? require('../../../secret/config.js')
+  .CARVIS_API : null;
 
-var login = function (username, password, userId) {
-  var path = baseURL + uberMethods.login.path;
-  var body = uberMethods.login.body(username, password);
-  var headers = uberMethods.login.headers();
+export const uberLogin = (req, res) => {
+  let username = req.body.username;
+  let password = req.body.password;
+  let userId = req.body.userId || null; // alexaUserId
 
-  userId = userId || null; // alexaUserId
+  let path = baseURL + login.path;
+  let body = login.body(username, password);
+  let headers = login.headers();
 
   fetch(path, {
       method: 'POST',
       headers: headers,
       body: JSON.stringify(body)
     })
-    .then(function (res) {
+    .then(res => {
       return res.json();
     })
-    .then(function (data) {
-      var response = uberMethods.login.responseMethod(data, password, userId);
+    .then(data => {
+      let response = login.responseMethod(data, password, userId);
 
       // POST THE USER DATA TO OUR RELATIONAL DATABASE
-      // var dbpostURL = 'http://' + APIserver + '/users/updateOrCreate';
-      var dbpostURL = 'http://localhost:8080/users/updateOrCreate';
-
-      console.log('response pre DB post', response);
+      let dbpostURL = 'http://' + APIserver + '/users/updateOrCreate';
```

Comment:
this is the only change i'd request for this PR, after which i'm down to merge @cpruijsen